server/client.go

```diff
@@ -1541,8 +1541,15 @@ func (c *client) processSub(argo []byte) (err error) {
 	} else if kind == CLIENT && !c.canSubscribe(string(sub.subject)) {
 		c.mu.Unlock()
 		c.sendErr(fmt.Sprintf("Permissions Violation for Subscription to %q", sub.subject))
-		c.Errorf("Subscription Violation - User %q, Subject %q, SID %s",
-			c.opts.Username, sub.subject, sub.sid)
+
+		var userInfo string
```

Comment:
I think better to make this a small function on client.

Reply:
Sounds good, just updated to use a small func on the client.